src/analysis/locate.mli

```diff
@@ -79,6 +79,7 @@ val from_string
   -> env:Env.t
   -> local_defs:Mtyper.typedtree
   -> pos:Lexing.position
+  -> context:Query_protocol.Locate_context.t option
```

Comment:
It looks like this function is always called with either a `namespaces` or a `context`, but not both. I think it would be cleaner to make this apparent in the type.

Reply:
Good idea, done